packages/esm-patient-chart-app/src/utils.ts

```diff
@@ -0,0 +1,6 @@
+import { LayoutType } from "@openmrs/esm-framework";
+
+export const isTablet = (layout: LayoutType) =>
```

Comment:
This is wrong. A tablet is not a phone.

It's rather `isNotDesktop`.

Reply:
changed to `isDesktop` instead: https://github.com/openmrs/openmrs-esm-patient-chart/pull/217/commits/f8c0876a14341877c1269814bc20bd7275ec7b94